src/storage/btree.c

```diff
@@ -6706,29 +6705,26 @@ btree_get_stats_with_AR_sampling (THREAD_ENTRY * thread_p, BTREE_STATS_ENV * env
 	  assert (header->node_level == 1);	/* BTREE_LEAF_NODE */
 #endif
 
-	  if (key_cnt > 0)
```

Comment:
The key_cnt of leaf node can be 0. If you remove this line, then the behavior is changed when the key_cnt is zero. Please explain more detail that the found is always false when the key_cnt of leaf node is zero. If you feel that the explanation of it is difficult then I recommand you to change just assert_release().

Reply:
I thought key_cnt was unconditionally bigger than zero, but I think I got confused with other issues. I turned it back.

Reply:
Please change the summary of this PR.